WalletWasabi/WabiSabi/Client/Banning/CoinPrison.cs

```diff
@@ -12,6 +12,8 @@ namespace WalletWasabi.WabiSabi.Client.Banning;
 
 public class CoinPrison
 {
+	private readonly int _absoluteMaximumLocalPrisonBanningTimeInDays = 31;
```

Comment:
@turbolay should I reduce this to 7 or lower?

Reply:
I think it should be low yes. The reason is that I'd like to use this to reply to user's support request about long banning time. But we can't say to the "Oh well try again in 30 days". So I believe it's better to make it to 3 days.

Reply:
Beware, this field is only used to determine if the banning time from the backend is too long and should not be saved.

The reason I choose `31` is to still display and do not override when someone gets filtered/kicked out by the _you know whom_.

If we change this to 3 days, then we override every banning time exceeding 3 days. (might result in less optimal coin selection much more often)

With these additional infos, are you still suggesting to reduce it to 3 days?